.github/workflows/storybook.yml

```diff
@@ -0,0 +1,79 @@
+name: npm-test
```

Comment:
@Akuukis based on the name and diff, I believe this file was staged accidentally?

Reply:
Yes, thanks for fixing it :pray: